### ISSUE TITLE
Atom 1.7.3 asciidoc-preview rendering broken

### ISSUE THREAD

**Issue by darshandsoni** [atom/atom#11749]
### Description
Atom 1.7.3: Previewing asciidoc works as expected. The preview no longer renders automatically though. Specifically, one cannot see an automatic update of new information render in the preview window until they close the preview window and reopen it.

### Steps to Reproduce

1. Write anything to an .adoc file
2. Preview with asciidoc-preview using the Ctrl + Shift + A (Cmd + Shift + A on Mac)
3. Preview will render as normal
4. Edit the information in the .adoc file
5. The preview pane will not reflect the updated information.

**Expected behavior:** Preview pane should live update the information. This is a working feature in Atom version 1.7.2

**Actual behavior:** Preview pane needed to be closed and reopened to reflect edits to the information.

### Versions
Atom version: 1.7.3
APM version: 1.9.2
Tested on Mac OS X 10.11.4

### Note
This is not a package issue because the asciidoc-preview hasn't received any updates since 2015-08-12.

**Comment by thomasjo** [comment]
@as-cii The only relevant change between 1.7.2 and 1.7.3 I can see that might be related to this is the text-buffer bump? I haven't attempted to repro this however, but figured I check with you.

Had a quick glance at the [asciidoc-preview code](https://github.com/asciidoctor/atom-asciidoc-preview/tree/master/lib), but I didn't immidiately see anything that uses private APIs or anything that would otherwise cause this regression...

**Comment by as-cii** [comment]
Thanks @thomasjo for the heads-up, and thank you @darshandsoni for reporting this issue.

Interestingly, I am not able to reproduce on **master** and neither on **1.7.3**. I noticed, however, that there's a setting in that package which explicitly enables re-rendering upon saving a file: can you confirm you haven't enabled that option? Also, could you try disabling every other package and make sure that none of them is influencing this wrong behaviour?

Thanks! 🙇

**Comment by darshandsoni** [comment]
Thanks for the follow up, @thomasjo and @as-cii.
I can confirm that I hadn't enabled that setting for re-rendering upon save. I tried disabling every other package to no avail either. 
I tested it out on my other machines running Fedora 23 and Ubuntu 16.04 and everything works normally. So I think it's just some weird botch on my Mac somewhere. Testing on another Mac soon... I will update you shortly...
